disnake/webhook/async_.py

```diff
@@ -1140,17 +1148,19 @@ def partial(
             "token": token,
         }
 
```

Comment:
Should we do an `isinstance(session, aiohttp.Session)` check here to be consistent with `SynchWebhook`?